x/accesscontrol/keeper/keeper.go

```diff
@@ -250,6 +250,26 @@ func BuildSelectorOps(accessOps []acltypes.AccessOperationWithSelector, senderBe
 				opWithSelector.Operation.IdentifierTemplate,
 				hexStr,
 			)
+		case acltypes.AccessOperationSelectorType_CONTRACT_REFERENCE:
+			// We use this to import the dependencies from another contract address
+			interContractAddress, err := sdk.AccAddressFromBech32(opWithSelector.Selector)
+			if err != nil {
+				return []acltypes.AccessOperationWithSelector{}, err
+			}
+			// TODO: add a circular dependency check here to ignore if we've already seen this contract/identifier in our reference chain
+			// for now, we will just pass in the same message body, this needs to be changed later though
+			// TODO: build new msgbody for the new contract execute / query msg in later milestone tasks
```

Comment:
we might want to add another TODO so that reference selectors of the referred contract are recursively evaluated

Reply:
This is already done since we call `GetWasmDependencyMapping` from within build selectors, but might be worth adding a test for this.